src/libcore/macros.rs

```diff
@@ -278,14 +278,14 @@ macro_rules! debug_assert_ne {
 ///
 /// // The previous method of quick returning Errors
 /// fn write_to_file_using_try() -> Result<(), MyError> {
-///     let mut file = try!(File::create("my_best_friends.txt"));
-///     try!(file.write_all(b"This is a list of my best friends."));
+///     let mut file = r#try!(File::create("my_best_friends.txt"));
```

Comment:
Is the change to the docs intended? If so, it might make sense to also explain or reference the raw literal syntax in the text above, as it may be a bit obscure for this rather prominent piece of documentation.

Reply:
This macro is deprecated (and further more `try!` doesn't even work on Rust 2018). Therefore it is not a prominent piece of documentation other than explaining that it is deprecated; but sure -- we can clarify the raw identifier syntax later (in a follow up PR -- which maybe @mark-i-m can fix up); the purpose `r#` fills here is to make code work on both editions (a good thing) so this is a strict improvement.

Reply:
Yes, the change was intended so that it works in both editions. Opened #56424

Reply:
Thanks!